addons/mrp/tests/test_warehouse_multistep_manufacturing.py

```diff
@@ -12,18 +12,14 @@ class TestMultistepManufacturingWarehouse(TestMrpCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        # Required for `uom_id` to be visible in the view
-        cls.env.user.groups_id += cls.env.ref('uom.group_uom')
-        # Required for `manufacture_steps` to be visible in the view
-        cls.env.user.groups_id += cls.env.ref('stock.group_adv_location')
-        # Create warehouse
-        cls.customer_location = cls.env['ir.model.data']._xmlid_to_res_id('stock.stock_location_customers')
-        warehouse_form = Form(cls.env['stock.warehouse'])
-        warehouse_form.name = 'Test Warehouse'
-        warehouse_form.code = 'TWH'
-        cls.warehouse = warehouse_form.save()
-
-        cls.uom_unit = cls.env.ref('uom.product_uom_unit')
+        with cls.sudo():
+            # TODO: decide if uom.group_uom should be assigned here or in common.py
```

Comment:
Should be in a helper function.
The way I see the different configuration. On top of each test I would call
```python
self._set_administration_groups()
self._set_multi_location_groups()
self._set_tracking_groups()
...

Reply:
Should be ready now, please check the recent commit :)